docs-content/installing.html.md.erb

```diff
@@ -59,33 +63,38 @@ This tab configures communication between the Service Broker and a Conjur applia
 
     <p class='note'><strong>Note:</strong> The Conjur Host credentials configured here are available only to PCF admins. They are not generally accessible to users in PCF.</p>
 
+  * **Conjur Version**: Defaults to 5. Select 4 for Enterprise Conjur v4. Select 5 for Enterprise Conjur v5 or Open Source Conjur (including an evaluation Hosted Conjur).
+
   * **PCF Conjur Policy Branch ID**: The fully-qualified Conjur Policy ID of the dedicated Conjur policy for PCF that you created. Leave blank to default to the root policy (not recommended).
 
-  * **Conjur Version**: Defaults to 5. Select 4 for Enterprise Conjur. Select 5 for Open Source Conjur (including an evaluation Hosted Conjur).
+  * **Preserve Conjur Policy on Deprovision**: By default, when the service broker is removed
```

Comment:
We should have a note here that this applies to Conjur EE v5+ and Conjur OSS v1+ only